webapp-django/crashstats/crashstats/static/crashstats/css/base/variables.less

```diff
@@ -1,39 +1,43 @@
+@import "photon-colors.less";
+
 @image-path: "/static/img";
 
-@base-font: Arial, Helvetica, sans-serif;
+@import "https://code.cdn.mozilla.net/fonts/fira.css";
```

Comment:
I think this is because `variables.less` are imported both in the `base` styles as well as the subpages. The base styles load the font, then the subpage's bundle also loads the font. I've moved this import to `fonts.less` which is imported only in the `base.less` file.